definitions/pipelines/rnaseq_star_fusion.cwl

```diff
@@ -14,6 +14,8 @@ inputs:
         type: string[]
     star_genome_dir:
         type: Directory
+    star_fusion_genome_dir:
+        type: Directory
```

Comment:
Can you add a quick "doc" line here (and to the star_genome_dir) line, giving a little info about what is expected to be in there?  (Or probably just linking to the wiki page for how to generate it!)